crates/flipperzero/src/macros.rs

```diff
@@ -2,22 +2,22 @@
 
 #[macro_export]
 macro_rules! print {
-    ($pat:expr, $($args:tt)*) => {{
+    ($pat:expr, $($args:tt)* $(,)*) => {{
```

Comment:
The `print!` and `println!` macros  should already handle trailing commas. For example, the code below compiles without issue. I believe this works because `,` is a valid token, thus is matched by `$($args:tt)*`.

```rust
use flipperzero::{println, print};

print!("Hello {}", 42,);
println!("Hello, Rust! {}", 42,);
```

Reply:
You're right. I've removed optional trailing comas from macros ending with repeated `tt`s (i.e. `print!` and `println!`).